### ISSUE TITLE
The validation_set is put into the training list.

### ISSUE THREAD

**Issue by Xun-Yang** [ChenJoya/2dtan#4]
I found that you put both the training set and validation set into the training list, which leads to a higher performance on the two datasets.

**Comment by ChenJoya** [comment]
In the original repo, it also does this. We test on the test set not on the val set. What is the problem??

**Comment by Xun-Yang** [comment]
Thanks for the reply. 

The problem is: In the original repo, its val set is just used to test the model (using model.eval(), the model parameters are not updated.) Please check the on_update() in the train.py (line 182) in the original repo.

In your repo, you directly use the val_set for training.

**Comment by ChenJoya** [comment]
Thank you for pointing out this. We will check it now. The result will be also updated.

**Comment by tujun233** [comment]
Dear @ChenJoya ,
I wanna make sure whether the result and code have been updated? I saw cfg.DATASETS.TRAIN still is ["tacos_train", "tacos_val"],
but in /tan/data/datasets/init.py line 36, dataset = datasets[0], does it means that the code only use tacos_train?
Thank you!
